print-ntp.c

```diff
@@ -516,3 +530,63 @@ p_poll(netdissect_options *ndo,
 		ND_PRINT(" (1/%us)", 1U << -poll_interval);
 }
 
+/* Prints an NTPv4 extension field */
+static void
+p_ntp_ef(netdissect_options *ndo, u_int type, u_int length, const u_char *ef_body)
+{
+	switch (type) {
```

Comment:
Why only 5 types?
(There are more:
https://www.iana.org/assignments/ntp-parameters/ntp-parameters.xhtml#ntp-parameters-3)

Reply:
> This switch could be replaced by a `struct tok ntp_ef_types`and a `tok2str()` call.

Ok, I did that.

> Why only 5 types? (There are more: https://www.iana.org/assignments/ntp-parameters/ntp-parameters.xhtml#ntp-parameters-3)

The other values in the table are not used by anything. The only known autokey implementation is using different values. This will be fixed by https://datatracker.ietf.org/doc/draft-ietf-ntp-update-registries/, but it's not finished yet. The tcpdump code can be updated later.